### DISCUSSION TITLE
PvP Arena Sets and Weapons to be Enabled for Mogging

### DISCUSSION THREAD

**Discussion by shaszun** [chromiecraft/chromiecraft#6281] [Suggestions]
Hello, can we get the PvP arena items to be mogable? These are not Horde or Alliance specific as previous PvP items and look very good. I would imagine anyone doing the work to get these would like to be able to mog their looks to other items.

**Comment by BeefSteaker** [comment]
You should be able to transmog them if you wait for the 2 hour refund window to expire
